allennlp_models/common/model_card.py

```diff
@@ -0,0 +1,267 @@
+import os
+import logging
+from dataclasses import dataclass
+from typing import Optional, Union, Mapping, Dict
+
+from allennlp.models import Model
+from allennlp.common.checks import ConfigurationError
+
+PRETRAINED_MODELS: Dict = {}
```

Comment:
By convention, having this name be all-uppercase means that we should consider it a constant, and it's fair game to access (but not change) from outside the module.

I think that's a defensible position to take, since this variable is definitely constant-ish, i.e., it should not change anymore after importing is done. I just want to point it out in case you didn't intend it that way.